### PR TITLE
Allow setting primary email `verified` flag

### PR DESCRIPTION
**Pull request by akikoskinen**
Adds possibility to control the `verified` flag of the primary email with the `createOrUpdateUserProfile` mutation.

### CODE REVIEW

File: profiles/schema.py [hunk 1]
```diff
@@ -1018,8 +1018,9 @@ def _handle_primary_email(profile, primary_email_input):
 
         profile.emails.exclude(pk=email.pk).filter(primary=True).update(primary=False)
 
-        if not email.primary:
+        if (not email.primary) or (email.verified is not verified):
```
Comment (mikkokeskinen) [inline profiles/schema.py:1021]: I would write this without the parentheses as that's the way it's usually done in python.

```python
if not email.primary or email.verified is not verified:
```

File: profiles/tests/test_gql_create_or_update_user_profile_mutation.py [hunk 2]
```diff
@@ -347,20 +347,25 @@ def test_create_primary_email_for_an_existing_profile(verified, user_gql_client)
 
 
 @pytest.mark.parametrize("old_is_primary", (True, False))
-def test_existing_primary_email_remains_when_trying_to_set_the_same_email_as_a_primary_email(
-    old_is_primary, user_gql_client
+@pytest.mark.parametrize("verified", [None, False, True])
```
Comment (mikkokeskinen) [inline profiles/tests/test_gql_create_or_update_user_profile_mutation.py:350]: Maybe always use tuples or always use lists in these?

**Review by mikkokeskinen — COMMENTED** [review]
Good stuff. Added two insignificant code style comments.

File: profiles/schema.py (hunk 1 above)
Comment (akikoskinen) [reply profiles/schema.py]: Changed.

File: profiles/tests/test_gql_create_or_update_user_profile_mutation.py (hunk 2 above)
Comment (akikoskinen) [reply profiles/tests/test_gql_create_or_update_user_profile_mutation.py]: Off course. Staying with tuples.

**Pull request merged**